docs/source/notes/ddp.rst

```diff
@@ -58,6 +58,11 @@ updated, and all models on different processes should be exactly the same.
             join=True)
 
     if __name__=="__main__":
+        # Environment variables which need to be
+        # set when using c10d's default "env"
+        # initialization mode.
+        os.environ["MASTER_ADDR"] = "localhost"
```

Comment:
does this need to be added into the `example` function so that all sub-processes can pick this up? Or will these env var settings persist and propagate to subprocesses?

Reply:
I think at least for `spawn` as in this example, the environment variables are indeed propagated to subprocess.